editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -122,8 +123,9 @@ export function useMaybeHighlightElement(): {
   )
 
   const maybeClearHighlightsOnHoverEnd = React.useCallback((): void => {
-    const { dispatch, dragging, resizing, selectionEnabled } = stateRef.current
-    if (selectionEnabled && !dragging && !resizing) {
+    const { dispatch, dragging, resizing, selectionEnabled, selectedViews } = stateRef.current
+
+    if (selectionEnabled && !dragging && !resizing && selectedViews.length > 0) {
```

Comment:
should this also check the length of the highlightedViews?

Reply:
Yep, it looks like this should be checking the length of `highlightedViews`, and not `selectedViews`

Reply:
yeah, I think both would be better

Reply:
To be clear here, this shouldn't be using both. It should only be using `highlightedViews`